cookieconsent/context_processors.py

```diff
@@ -0,0 +1,8 @@
+
+
+def cookies_accepted(request):
+    cookieconsent = False
+    if request.COOKIES.get('cookieconsent', 'decline') == 'accept':
+        cookieconsent = True
+
+    return {'COOKIES_ACCEPTED': cookieconsent}
```

Comment:
COOKIES_ACCEPTED context does not seem to be used. or it will be used later as mentioned in the issue description.